### DISCUSSION TITLE
Anti aliased Panel Sprite

### DISCUSSION THREAD

**Discussion by HelperMuzan** [4ian/GDevelop#2728] [Technical questions about the codebase]
It might not be some kind of bug or something but the Panel Sprite or 9 patch gets anti aliasing when moving my sprite using some behavior and centering the camera upon him.

**Comment by tristanbob** [comment]
Can you share a screen recording or a game project that we can use to test and duplicate?

**Reply by HelperMuzan** [reply]
Where can I share it?

**Comment by Bouh** [comment]
Hello @Mixen-me 

Thank you for this feedback, would it be possible to share your feedback with the users on the forum?

The bug report (Which is not one because you are here on a discission) you opened contains too little detail, share your feedback on the forum, aggregate as much information as possible, tests, examples, captures, everything is useful.

We want to know first how this impact users.

**Comment by HelperMuzan** [comment]
This might not be clear but you can see how the "9-Patch" gets
blurry/anti-aliased in realtime.

https://user-images.githubusercontent.com/85349700/121665213-0587fc00-cac8-11eb-8f06-0fb044d5d5c8.mp4

These are images...
What it should be:
![should be](https://user-images.githubusercontent.com/85349700/121666806-6106b980-cac9-11eb-9601-c8ee7a13b0ef.PNG)
What it is:
![it is](https://user-images.githubusercontent.com/85349700/121666893-78de3d80-cac9-11eb-903a-86779177cb74.PNG)

I have seen this happening with pixel perfect game projects. This only happens when you move your character slightly, pressing and releasing the move key very fast.
